app/presenters/deposit_title_presenter.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# A helper for displaying a title for a Work or Collection
+class DepositTitlePresenter
+  def self.show(deposit)
+    if deposit.instance_of?(WorkVersion)
+      deposit&.title.presence || Settings.h2.no_title
+    elsif deposit.instance_of?(CollectionVersion)
+      deposit&.name.presence || Settings.h2.no_title
```

Comment:
it was being done in the `WorkTitlePresenter` before -- not sure why though.  I removed, and seems to work in localhost but not sure if I've hit all the edge cases of new works (though again, I don't **think** it will be nil).